### PR TITLE
Webops 972/cw stats

### PR DESCRIPTION
**Pull request by thepwagner**
Functional tested on a spacel AMI via docker build: dat agility.
This is ready for functional testing as an AMI.

Docker logs bits tested via hack (just re-using `DeployLogGroup`); will pick up the spacel-provision support.

### CODE REVIEW

File: src/spacel/agent/mixin/cloudwatch_logs.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+class SetupCloudWatchLogs(BaseMixinWriter):
+    def __init__(self, systemd, meta):
+        super(SetupCloudWatchLogs, self).__init__(systemd)
+        self._meta = meta
+
+    def logs(self, manifest):
+        self._docker_logs(manifest)
+        # TODO: self._journald_logs ?
```
Comment (thepwagner) [inline src/spacel/agent/mixin/cloudwatch_logs.py]: Is this feature still worth it?

The annoying bit is getting https://github.com/saymedia/journald-cloudwatch-logs baked into the AMI: go binary without a PPA.

Comment (DeviaVir) [inline src/spacel/agent/mixin/cloudwatch_logs.py]: We already get the valuable logs into `DeployLogGroup`? In that case allowing journald separately seems like overkill.

File: src/spacel/aws/meta.py [hunk 2]
```diff
@@ -1,29 +1,35 @@
...
+
 from spacel.aws.helpers import read_file
 
-INSTANCE_ID_URL = 'http://169.254.169.254/latest/meta-data/instance-id'
-PLACEMENT_URL = ('http://169.254.169.254/latest/meta-data/placement/'
-                 'availability-zone')
-USER_DATA_URL = 'http://169.254.169.254/latest/user-data'
-BLOCK_DEVICES_URL = ('http://169.254.169.254/latest/meta-data/'
-                     'block-device-mapping/')
+INSTANCE_DATA = 'http://instance-data/latest/'
```
Comment (DeviaVir) [inline src/spacel/aws/meta.py]: _noice_

**Comment by thepwagner** [comment]
+2 for no journald: if you _really_ want to know what `ntpd` is up to you can log in.

**Pull request merged**